command/v7/set_space_role_command.go

```diff
@@ -16,7 +16,7 @@ type SetSpaceRoleCommand struct {
 	Args            flag.SpaceRoleArgs `positional-args:"yes"`
 	IsClient        bool               `long:"client" description:"Assign a space role to a client-id of a (non-user) service account"`
 	Origin          string             `long:"origin" description:"Indicates the identity provider to be used for authentication"`
-	usage           interface{}        `usage:"CF_NAME set-space-role USERNAME ORG SPACE ROLE\n   CF_NAME set-space-role USERNAME ORG SPACE ROLE [--client]\n   CF_NAME set-space-role USERNAME ORG SPACE ROLE [--origin ORIGIN]\n\nROLES:\n   SpaceManager - Invite and manage users, and enable features for a given space\n   SpaceDeveloper - Create and manage apps and services, and see logs and reports\n   SpaceAuditor - View logs, reports, and settings on this space\n   SpaceSupporter - (Beta role, behavior will change) TEMPLATE MESSAGE"`
+	usage           interface{}        `usage:"CF_NAME set-space-role USERNAME ORG SPACE ROLE\n   CF_NAME set-space-role USERNAME ORG SPACE ROLE [--client]\n   CF_NAME set-space-role USERNAME ORG SPACE ROLE [--origin ORIGIN]\n\nROLES:\n   SpaceManager - Invite and manage users, and enable features for a given space\n   SpaceDeveloper - Create and manage apps and services, and see logs and reports\n   SpaceAuditor - View logs, reports, and settings on this space\n   SpaceSupporter [Beta role, subject to change] - Manage apps and services, and see logs and reports"`
```

Comment:
I like this message, but I'm starting to overthink a bit. Thoughts on this wording for the first half:

Manage app lifecycle and service bindings

Open to changing this though if this isn't what app lifecycle means to you.

Reply:
Updated this in the newest commit